app/editor/src/features/content/form/styled/ContentForm.tsx

```diff
@@ -20,6 +20,39 @@ export const ContentForm = styled.div`
     }
   }
 
+  .src-cpy {
+    border: solid 1px #606060;
+    border-left: none;
+    border-top-right-radius: 0.25em;
+    border-bottom-right-radius: 0.25em;
+  }
+
+  .disabled {
+    color: ${(props) => props.theme.css.lightVariantColor} !important;
```

Comment:
How come we need to use `!important`?  This almost always indicates an implementation issue.